packages/firestore/src/local/simple_db.ts

```diff
@@ -66,7 +66,14 @@ export class SimpleDb {
       // suggests IE9 and older WebKit browsers handle upgrade
       // differently. They expect setVersion, as described here:
       // https://developer.mozilla.org/en-US/docs/Web/API/IDBVersionChangeRequest/setVersion
-      const request = window.indexedDB.open(name, version);
+
+      let request: IDBOpenDBRequest;
+      // Support Web Workers, which don't have `window` global variable.
+      if (indexedDB !== undefined) {
+        request = indexedDB.open(name, version);
+      } else {
+        request = window.indexedDB.open(name, version);
```

Comment:
Since `window` is _usually_ the global object and we've already checked for the `indexedDB` global directly, is there a case where we will ever hit this?

Maybe we have a fake `window` object in node tests or something?  Do you know?